src/routes/engineRoutes.ts

```diff
@@ -0,0 +1,8 @@
+import express from "express";
```

Comment:
This route could be in the project routes since it is data that comes from the same document and it does not make sense to make a route index for only one.

Reply:
Fixed by adding the code to projectRoutes